From 113a9cea0ce4b606c46eb5842dd9d2c8c975a666 Mon Sep 17 00:00:00 2001
From: John Cox <jc@kynesim.co.uk>
Date: Mon, 2 Oct 2023 15:15:13 +0100
Subject: [PATCH] dts/bcm2712: Add iommu to rpivid

Add iommu to rpivid so it can cope with scatter/gather

Signed-off-by: John Cox <jc@kynesim.co.uk>
---
 arch/arm/boot/dts/broadcom/bcm2712.dtsi | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/boot/dts/broadcom/bcm2712.dtsi
+++ b/arch/arm/boot/dts/broadcom/bcm2712.dtsi
@@ -1156,6 +1156,7 @@
 
 			clocks = <&firmware_clocks 11>;
 			clock-names = "hevc";
+			iommus = <&iommu2>;
 			status = "disabled";
 		};
 
